docs/DeveloperGuide.md

```diff
@@ -1,37 +1,1036 @@
+`/exercise` and call the constructor of `SearchCommand` class by passing relevant parameters related to search exercise
+to the constructor. The created `SearchCommand` object is returned by the `Parser#createSearchCommand(String userInput)`
+method to `Parser#parseUserInput(String userInput)` method, and finally returned by
+`Parser#parseUserInput(String userInput)` method to `WerkIt#startContinuousUserPrompt()` method. The search command will
+be executed in `WerkIt#startContinuousUserPrompt()`. And based on the `<keywords>` specified by the user, the output
+will either be a list of matching exercises or 'Sorry, no matching exercise found' if the user has entered the command
+correctly.
+
+The following sequence diagram illustrates how the `search /exercise` command works in greater detail:
+
+![Search Exercise Sequence Diagram](uml/sequenceDiagrams/images/searchExercise.png)
+
+#### Search For Workout
+Format: `search /workout <keywords>`
+
+The `Parser#createSearchCommand(String userInput)` method will further evaluate the user input
+`/workout` and call the constructor of `SearchCommand` class by passing relevant parameters to the constructor.
+The created `SearchCommand` object is returned by the `Parser#createSearchCommand(String userInput)`
+method to `Parser#parseUserInput(String userInput)` method, and finally returned by
+`Parser#parseUserInput(String userInput)` method to `WerkIt#startContinuousUserPrompt()` method. The search command will
+be executed in `WerkIt#startContinuousUserPrompt()`. And based on the `<keywords>` specified by the user, the output
+will either be a list of matching names of workout or 'Sorry, no matching workout found' if the user has entered the command
+correctly.
+
+The following sequence diagram illustrates how the `search /workout` command works in greater detail:
+
+![Search Exercise Sequence Diagram](uml/sequenceDiagrams/images/searchWorkout.png)
+
+#### Search For Plan
+Format: `search /plan <keywords>`
+
+The `Parser#createSearchCommand(String userInput)` method will further evaluate the user input
+`/plan` and call the constructor of `SearchCommand` class by passing relevant parameters to the constructor.
+The created `SearchCommand` object is returned by the `Parser#createSearchCommand(String userInput)`
+method to `Parser#parseUserInput(String userInput)` method, and finally returned by
+`Parser#parseUserInput(String userInput)` method to `WerkIt#startContinuousUserPrompt()` method. The search command will
+be executed in `WerkIt#startContinuousUserPrompt()`. And based on the `<keywords>` specified by the user, the output
+will either be a list of matching names of plan or 'Sorry, no matching plan found' if the user has entered the command
+correctly.
+
+The following sequence diagram illustrates how the `search /plan` command works in greater detail:
+
+![Search Exercise Sequence Diagram](uml/sequenceDiagrams/images/searchPlan.png)
```

Comment:
<img width="921" alt="Screenshot 2022-03-31 at 10 53 28" src="https://user-images.githubusercontent.com/60871570/160966988-f567e2e3-ec24-453c-b06d-cedb309b76f4.png">
Perhaps you could remove the boxes at the bottom to follow the standard